src/components/config/config.ts

```diff
@@ -126,6 +126,14 @@ export async function setAlwaysCleanUp(): Promise<void> {
     await setConfigValue(AUTO_CLEANUP_DEBUG_KEY, true);
 }
 
+// Use WSL on Windows
+
+const USE_WSL_KEY = "useWsl";
```

Comment:
We usually hyphenate rather than camel-case config keys I think - suggest

```suggestion
const USE_WSL_KEY = "use-wsl";
```

Reply:
done.